core/src/main/java/org/opensearch/sql/planner/logical/LogicalValues.java

```diff
@@ -14,16 +13,14 @@
 import org.opensearch.sql.expression.LiteralExpression;
 
 /**
- * Logical operator which is a sequence of literal rows (like a relation).
- * Basically, Values operator is used to create rows of constant literals
- * "out of nothing" which is corresponding with VALUES clause in SQL.
- * Mostly all rows must have the same number of literals and each column should
- * have same type or can be converted implicitly.
- * In particular, typical use cases include:
```

Comment:
can we ignore spotless here?
or add list tags...

Reply:
Fixed